src/components/Navbar.js

```diff
@@ -1,22 +1,27 @@
-import React from "react";
-import styled from "styled-components";
-import { Navbar, Nav, Button } from "react-bootstrap";
+import React from 'react'
+import styled from 'styled-components'
+import { Navbar, Nav, Button } from 'react-bootstrap'
 
-export default (props) =>
-{
-  return <CustomNavbar sticky="top" collapseOnSelect expand="md">
-    <Navbar.Brand href="/" style={{ color: '#30323D', fontSize: '1.9rem' }}>SG VERIFACT</Navbar.Brand>
-    <Navbar.Toggle aria-controls='responsive-navbar-nav' />
-    <Navbar.Collapse className="justify-content-end" id='responsive-navbar-nav' style={{ backgroundColor: 'white', margin: '1.1rem 0' }}>
-      <Nav className='mr-auto'></Nav>
-      <CustomNav>
-        <Link href="login">Log In</Link>
-        <Link href="signup">Sign Up</Link>
-        <CustomButton href="/askquestion">Ask a Question</CustomButton>
-      </CustomNav>
-    </Navbar.Collapse>
-  </CustomNavbar >
-};
+export default (props) => {
+  return (
+    <CustomNavbar sticky='top' collapseOnSelect expand='md'>
+      <Navbar.Brand href='/' style={{ color: '#30323D', fontSize: '1.9rem' }}>SG VERIFACT</Navbar.Brand>
+      <Navbar.Toggle aria-controls='responsive-navbar-nav' />
+      <Navbar.Collapse id='responsive-navbar-nav' style={{ backgroundColor: 'white' }}>
+        <Nav className='mr-auto' />
+        <Nav style={{ alignItems: 'center' }}>
+          <Nav.Link style={{ color: '#30323D' }} href='login'>Log In</Nav.Link>
+          <Nav.Link style={{ color: '#30323D' }} href='register'>
+            Register
```

Comment:
I've asked several times why we are changing this to `Register`. Unless there's a specific reason, I'd like to change it back to `Sign Up` to avoid divergence from the design.